AlfrescoApp/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -622,26 +622,26 @@
 
 // Cell
 "tasks.cell.taskcompleted" = "%@ completed this task";
-"tasks.cell.tasknotcompleted" = "%@ has not completed this task";
+"tasks.cell.tasknotcompleted" = "%@ hasn't completed this task";
 // Process definition titles - these match workflow-message.properties in the repository
 "tasks.process.activitiAdhoc" = "New Task";
-"tasks.process.activitiParallelReview" = "Send File(s) For Review";
-"tasks.process.activitiReview" = "Review And Approve";
+"tasks.process.activitiParallelReview" = "Send File(s) for Review";
+"tasks.process.activitiReview" = "Review and Approve";
```

Comment:
@andyhealey26 Just to double-check the case change here, as we tend to use Title Case throughout our iOS app.